profiles/base/package.use.mask

```diff
@@ -6,6 +6,11 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Jimi Huotari <chiitoo@gentoo.org> (2021-04-28)
+# Known to cause issues with some scripts, while fixing others.
```

Comment:
From your description it would seem odd to mask it. Notwithstanding the patch tipping the scales in one or the other way, at least it sounds like there have been existing issues that haven't lead to the flag be masked yet.

Maybe just add an `ewarn` in `pkg_postinst` and wait for the reports to be filed?

Reply:
Yep, done.